patroni/api.py

```diff
@@ -65,15 +68,15 @@ def check_auth_header(self):
 
     def _write_status_response(self, status_code, response, options=False):
```

Comment:
'options' parameter should have a less confusing name, how about 'method_OPTIONS'?